application/comit_node/src/http_api/rfc003/swap.rs

```diff
@@ -256,11 +284,32 @@ struct SwapCommunication {
 
 #[derive(Debug, Serialize)]
 struct LedgerState {
-    current_state: String,
-    htlc_location: Option<String>,
-    fund_tx: Option<String>,
-    redeem_tx: Option<String>,
-    refund_tx: Option<String>,
+    current_state: HtlcState,
+    htlc_location: Option<Value>,
+    fund_tx: Option<Value>,
+    redeem_tx: Option<Value>,
+    refund_tx: Option<Value>,
+}
+
+impl Default for LedgerState {
+    fn default() -> Self {
+        Self {
+            current_state: HtlcState::NotDeployed,
```

Comment:
If you make `NotDeployed` the `Default` of `HtlcState`, you can `#[derive(Default)]` here.